### ISSUE TITLE
Get all RSS tags

### ISSUE THREAD

**Issue by hassanasad** [Anonyfox/meteor-scrape#19]
Hi,

I wanted to know how to get all the tags present in the RSS feed? Currently the `Scrape.feed` pull only selected tags.

**Comment by Anonyfox** [comment]
May you provide an example feed an the desired results you want?

**Comment by hassanasad** [comment]
Oh sorry, sure.

https://clinicaltrials.gov/ct2/results/rss.xml?rcv_d=&lup_d=14&term=childhood+cancer&show_rss=Y&sel_rss=mod14&count=10000

I want to be able to get all the XML tags in here including `<guid>`

Also I want to parse this RSS too : https://clinicaltrials.gov/ct2/show/NCT01073735?term=childhood+cancer&rank=1&sel_rss=mod14&displayxml=true

If your package cannot do this (because its using google feed API), can you please suggest any javascript service/package that can?

The only method i can think of to solve this problem is to create an API in PHP that converts RSS to JSON (but i don't want to do it unless its last resort).

P.S: This is my first assignment in a javascript based framework so my knowledge is pretty limited :)

**Comment by Anonyfox** [comment]
No, this package _isn't_ using the google feeds api, and I don't want to. As you can see here: https://github.com/Anonyfox/meteor-scrape/blob/master/lib/parse-feed.coffee.md , the parsing of the RSS XML is very simple and straightforward (when you know how to use jQuery you basically know how to get what you want). Also, it's written in literate coffeescript to increase the readability. 

I'd gladly accept a pull request that adds the feature you want, should be a one-liner to add `<guid>` and you improve this open source project for everyone. :)

**Comment by hassanasad** [comment]
Oh. Well thanks for pointing me in this direction.

However, what i was hoping was to get ALL the data available in the feed to be converted into JSON :)

Like in my second example of the feed i sent ( https://clinicaltrials.gov/ct2/show/NCT01073735?term=childhood+cancer&rank=1&sel_rss=mod14&displayxml=true ) it uses very different RSS tags.

Now i might be able to add a line or two but i am pretty sure conversion of an entire feed into JSON objects is little out of my skill set :)

**Comment by Anonyfox** [comment]
This library is intended to extract the _essential_ data pieces from feeds/websites, so if you basically want to have _everything_ in just another data format, have a look at the lib's mentioned in http://stackoverflow.com/questions/1773550/convert-xml-to-json-and-back-using-javascript :)
